### DISCUSSION TITLE
I need help to enter again dev mode

### DISCUSSION THREAD

**Discussion by jr-isp** [dnschneid/crouton#5095] [Q&A]
Hello everyone,

I bought this laptop several years ago **[XE500C21]** and for the most time I used ubuntu xenial on it without much of a trouble (other than very old versions of programs), all of this using crouton. As you probably know, crouton only runs in dev mode.

I finally decided to upgrade to ubuntu 18 (bionic beaver), because I read online it might be possible. Now I tried everything software-related and it just didn't work. Here I would like to add that before crouton I tried to completely remove ChromeOS, but I was not able to do this because of the screw protecting changes to the firmware (I don't know where this screw is, no picture of this model actually showed it. Only for other models). While I was trying to find the screw in the hopes it might be what I need to upgrade it, I removed the battery.

I finally was not able to find it, but when I turned it back on, it wouldn't run on dev mode. It does run as for default (ChromeOS), but I need dev mode to use crouton. It uses Chrome 58, it does not support anything anymore... it's pretty much useless as it is right now.

Please, help me get back dev mode, at least in xenial I can actually use the laptop!

**Comment by DennisLfromGA** [comment]
@jr-isp,

I responded to you on the chromeos Discord server but I'll mention it again here.

> It uses Chrome 58  

Well, you could update to [M93](<https://cros.tech/device/winky/>) but I wouldn't waste my time, unless it's needed to enter dev mode.  
> Samsung Chromebook XE500C21-AZ2US Laptop Computer, 1.66 GHz , 2GB DDR3 Ram, 16GB SSD  

That's a pitifully old & inadequate device but you can find the wrtie protect screw here:  
- [Installing Winky](<https://wiki.galliumos.org/Installing/Winky>)  



-**DennisLfromGA**
